src/App.js

```diff
@@ -1,7 +1,10 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-
+  return (
+    <div className="calculator-container"><Calculator /></div>
+  );
 }
```

Comment:
- [ ] From the requirements of this project we are expected to use class-based components instead of functional-based components. Please let's refactor this component to a class-based component. By doing this, we get closer to approval.

Reply:
Thanks for reaching out and telling me my changes. I have successfully implemented them 👍